.config/.prettierignore

```diff
@@ -18,4 +19,4 @@
 !test-strategy.md
 **/js/_space-test/**
 js/app/test/gradio_cached_examples/**
-
+**/theme.css
```

Comment:
I intended to ignore that file and `js/app/src/lite/theme.css`, as `generate_theme.py` has been used to create it as well since  #4788.
But yes, specifying the exact paths would be better. Will change it.